### PR TITLE
Hotfix: Update Students and Recent Graduates Careers page

### PR DESCRIPTION
**Pull request by Scotchester**
Brandeis Honors Attorney Program application window has closed; DFA window is about to open.

It was requested to publish these changes by COB Wednesday.

## Changes

- Updated copy on Students and Recent Graduates Careers apge

## Testing

1. Pull branch
2. Run server
3. Visit <http://localhost:8000/about-us/careers/students-and-graduates/>

## Review

- @sarahsimpson09 
- @rosskarchner 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by higs4281** [comment]
👍 looks good.

**Comment by sebworks** [comment]
👍

File: cfgov/jinja2/v1/about-us/careers/students-and-graduates/index.html [hunk 1]
```diff
@@ -73,8 +69,14 @@ <h3 class="h5">
         </div>
         <div class="media_body">
             <h2>CFPB Summer Internship</h2>
-            <h3 class="h5">Look for applications here starting Fall 2016</h3>
-            <p>
+            <div class="h5">
+                Applications open 10/20/16
+            </div>
+            <a class="jump-link jump-link__download"
+               href="http://files.consumerfinance.gov/f/documents/cfpb_RecruitmentFlyer_SummerInternship.pdf">
```
Comment (vccabral) [inline cfgov/jinja2/v1/about-us/careers/students-and-graduates/index.html]: why did we move toward hard coding?

Comment (Scotchester) [inline cfgov/jinja2/v1/about-us/careers/students-and-graduates/index.html]: We didn't; we're moving away from it. This is one of the last vestiges. Just not quite ready for it to go into Wagtail, and this update is urgent.

**Comment by Scotchester** [comment]
Screenshot for @sarahsimpson09:

screen shot 2016-09-14 at 11 31 26

**Comment by sarahsimpson09** [comment]
:+1: The stakeholder has signed off as well. Ready to publish!

**Comment by Scotchester** [comment]
@rosskarchner or whomever else is empowered: please release this hotfix by the end of the day.

**Comment by Scotchester** [comment]
Let me know if I can do anything to help.

**Comment by sarahsimpson09** [comment]
Thanks for getting to this so quickly, @Scotchester!

**Comment by rosskarchner** [comment]
I've confirmed with @Scotchester, this is live!

**Comment by Scotchester** [comment]
Thank you, @rosskarchner!

**Pull request merged**